etl/steps/data/garden/hmd/2024-11-19/hfd.meta.yml

```diff
@@ -0,0 +1,288 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  others:
+    bo_1: |-
+      <% if birth_order == '1' %>
+       first
+      <%- elif birth_order == '2' %>
+       second
+      <%- elif birth_order == '3' %>
+       third
+      <%- elif birth_order == '4' %>
+       fourth
+      <%- elif birth_order == '5p' %>
+       fifth (or greater)
+      <%- else %>
+      <% endif %>
+    title: |-
+      <% if birth_order == 'total' %>
+      << title >> - Total
+      <%- else %>
+      << title >> - Birth order: << birth_order >>
+      <%- endif %>
+  common:
+    presentation:
+      topic_tags:
+        - Fertility Rate
+    display:
+      numDecimalPlaces: 2
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+  title: Human Fertility Database
+
+tables:
+  period:
+    variables:
+      tfr:
+        title: |-
+          <% set title = "Period total fertility rates" %>
+          {definitions.others.title}
+        description_short: |-
+          The average number of{definitions.others.bo_1} children a woman would have in her lifetime if she experienced the fertility rates of a specific year.
+        unit: "births per woman"
+        description_key:
+          - Assumes current age-specific fertility rates remain constant throughout a woman's lifetime.
+          - Does not account for potential changes in social, economic, or health conditions that could affect fertility rates.
+
+      tfr40:
+        title: |-
+          <% set title = "Period total fertility rates by age 40" %>
+          {definitions.others.title}
+        description_short: |-
+          The average number of{definitions.others.bo_1} children a woman would have by age 40 if she experienced the fertility rates of a specific year.
+        unit: "births per woman"
+        description_key:
+          - Useful for understanding early and mid-reproductive age fertility patterns.
+
+      adjtfr:
+        title: |-
+          <% set title = "Tempo-adjusted total fertility rates" %>
+          {definitions.others.title}
+        description_short: |-
+          <% if birth_order == 'total' %>
+          The total fertility rate adjusted to account for delays or advances in childbearing.
+          <%- else %>
+          The total fertility rate for{definitions.others.bo_1} births adjusted to account for delays or advances in having a{definitions.others.bo_1} child.
+          <%- endif %>
+        unit: "births per woman"
+        description_key:
```

Comment:
You can also define Total fertility rate (TFR) somewhere and then use TFR in the following texts

Reply:
I don't fully understand your suggestion here. There are various TFR types, each with a different definition. E.g. for this one, `adjtfr` it needs a slight variant of the original `tfr` definition.